src/main/resources/io/seqera/wave/build-view.hbs

```diff
@@ -117,6 +117,13 @@
                 <td><a href="{{scan_url}}">Image scan report</a></td>
             </tr>
         {{/if}}
+
+        {{#if build_success}}
+            {{#if build_conda_lock_url }}
+                <td>Conda lockfile</td>
+                <td><a href="{{build_conda_lock_url}}">Conda lockfile</a></td>
+            {{/if}}
```

Comment:
let's render the conda lock text after the conda environment block

Reply:
do you mean after conda file?
https://github.com/seqeralabs/wave/blob/11816def0fea5cd4c0a24a6255a027e1752a0765/src/main/resources/io/seqera/wave/build-view.hbs#L132-L136

Reply:
Yes, using a separate code block to show the lock file

Reply:
should we show the conda lockfile text in build page or just download link?

Reply:
ok same like logs, where we truncate after max size

should i put the same max size for condalock as logs?

Reply:
so only test block and no download link?

Reply:
yep

Reply:
done
<img width="989" alt="Screenshot 2024-10-03 at 16 05 04" src="https://github.com/user-attachments/assets/b44c4abc-a56d-47b9-be0e-5fedcb2d5f69">

Reply:
Nice! any chance to add a copy & past action like in github code blocks 

<img width="812" alt="Screenshot 2024-10-03 at 10 23 27" src="https://github.com/user-attachments/assets/10223fc7-ae95-453c-8e76-39eff128f755">

Reply:
sorry missed it, will check it out